src/Tizen.TV.UIControls.Forms/Renderer/TVESPlayerImpl.cs

```diff
@@ -0,0 +1,420 @@
+            }
+        }
+
+        IVideoOutput VideoOutput
+        {
+            get { return _videoOutput; }
+            set
+            {
+                if (TargetView != null)
+                    TargetView.PropertyChanged -= OnTargetViewPropertyChanged;
+                if (OverlayOutput != null)
+                    OverlayOutput.AreaUpdated -= OnOverlayAreaUpdated;
+
+                _videoOutput = value;
+
+                if (TargetView != null)
+                    TargetView.PropertyChanged += OnTargetViewPropertyChanged;
+                if (OverlayOutput != null)
+                    OverlayOutput.AreaUpdated += OnOverlayAreaUpdated;
+            }
+        }
+
+        VisualElement TargetView => VideoOutput?.MediaView;
+
+        IOverlayOutput OverlayOutput => TargetView as IOverlayOutput;
+
+        void ApplyDisplay()
+        {
+            if (VideoOutput == null)
+            {
+                esPlayer.SetDisplay(null);
+            }
+            else
+            {
+                esPlayer.SetDisplay(TVForms.UIControls.MainWindowProvider());
+                OverlayOutput.AreaUpdated += OnOverlayAreaUpdated;
+                ApplyOverlayArea();
+            }
+        }
+
+        void OnOverlayAreaUpdated(object sender, EventArgs e)
+        {
+            ApplyOverlayArea();
+        }
+
+        async void OnTargetViewPropertyChanged(object sender, global::System.ComponentModel.PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName == "Renderer")
+            {
+                if (Platform.GetRenderer(sender as BindableObject) != null && AutoPlay)
+                {
+                    await Start();
+                }
+                else if (Platform.GetRenderer(sender as BindableObject) == null && AutoStop)
+                {
+                    Stop();
+                }
+            }
+        }
+
+        async void ApplyOverlayArea()
+        {
+            try
+            {
+                if (OverlayOutput.OverlayArea.IsEmpty)
+                {
+                    esPlayer.SetDisplayMode(AspectMode.ToTVDisplayMode());
+                }
+                else
+                {
+                    var bound = OverlayOutput.OverlayArea.ToPixel();
+                    var renderer = Platform.GetRenderer(TargetView);
+                    if (renderer is OverlayViewRenderer)
+                    {
+                        var parentArea = renderer.NativeView.Geometry;
+                        if (parentArea.Width == 0 || parentArea.Height == 0)
+                        {
+                            await Task.Delay(1);
+                            parentArea = renderer.NativeView.Geometry;
+                        }
+                        bound = parentArea;
+                    }
+                    var roiBound = bound.ToMultimedia();
+                    esPlayer.SetDisplayRoi(roiBound.X, roiBound.Y, roiBound.Width, roiBound.Height);
+                }

```

Comment:
Need to refactoring, ESPlayer is different from Tizen.Player on `DisplayMode`, 

Tizen.Player explicitly has ROI mode, in this mode video was displayed on roi area and if not ROI mode, video displayed on window area, 
But, ESPlayer has no ROI mode, So, it seem to no way to return full screen after call `SetDisplayRoi`